src/bidiMapper/modules/cdp/CdpTarget.ts

```diff
@@ -167,6 +167,10 @@ export class CdpTarget {
         this.#cdpClient.sendCommand('Page.enable', {
           enableFileChooserOpenedEvent: true,
         }),
+        this.#cdpClient.sendCommand('Browser.setDownloadBehavior', {
```

Comment:
Do we need this for every CDP target, isn't the browser domain only on the Browser Target?

Reply:
indeed. Fixed